Revit_Core_Engine/Convert/Revit/FromRevit/View.cs

```diff
@@ -0,0 +1,135 @@
+        /***************************************************/
+
+        [Description("Converts a Revit View to BH.oM.Adapters.Revit.Elements.View.")]
+        [Input("revitView", "Revit View to be converted.")]
+        [Input("settings", "Revit adapter settings to be used while performing the convert.")]
+        [Input("refObjects", "Optional, a collection of objects already processed in the current adapter action, stored to avoid processing the same object more than once.")]
+        [Output("view", "BH.oM.Adapters.Revit.Elements.View resulting from converting the input Revit View.")]
+        public static oM.Adapters.Revit.Elements.View ViewFromRevit(this View revitView, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null)
+        {
+            settings = settings.DefaultIfNull();
+
+            oM.Adapters.Revit.Elements.View view = refObjects.GetValue<oM.Adapters.Revit.Elements.View>(revitView.Id.IntegerValue);
+            if (view != null)
+                return view;
+
+            /*1. Transfer ViewFilters and corresponding OverrideGraphicSettings into ViewFilterWithOverrides Objects - via STREAMS */
+            List<ViewFilterWithOverrides> filtersWithOverrides = revitView.GetFilters().ToDictionary<ElementId, ViewFilter, OverrideGraphicSettings>
+                                                        (elId => Convert.ViewFilterFromRevit((ParameterFilterElement)revitView.Document.GetElement(elId)),
+                                                         elId => overrideGraphicSettingsFromRevit(revitView, revitView.GetFilterOverrides(elId))).ToList()
+                                                         .Select(kvp => new ViewFilterWithOverrides { Filter = kvp.Key, Overrides = kvp.Value }).ToList();
+
+            /*2. Create BHoM View Object with Name and FilterWithOverrides objects */
+            view = new oM.Adapters.Revit.Elements.View { Name = revitView.Name, FiltersWithOverrides = filtersWithOverrides };
+
+            //Set identifiers, parameters & custom data
+            view.SetIdentifiers(revitView);
+            view.CopyParameters(revitView, settings.MappingSettings);
+            view.SetProperties(revitView, settings.MappingSettings);
+
+            refObjects.AddOrReplace(revitView.Id, view);
+            return view;
+        }
+
+        /***************************************************/
+
+        private static OverrideGraphicSettings overrideGraphicSettingsFromRevit(this View element, Autodesk.Revit.DB.OverrideGraphicSettings revitOverrides)
+        {
+            // Initialize BHoM OverrideGraphicsSettings object
+            OverrideGraphicSettings overrideGraphicsSettings = new OverrideGraphicSettings();
+
+            // Convert COLORS 
+            overrideGraphicsSettings.LineColor = System.Drawing.Color.FromArgb(revitOverrides.CutLineColor.Red, revitOverrides.CutLineColor.Green, revitOverrides.CutLineColor.Blue);
```

Comment:
You need to check if `revitOverrides.CutLineColor.IsValid` first, otherwise it crashes in case of no color override applied

Reply:
As discussed live, need to avoid mixing dash & dot, so if (contains "Dash" but not "Dot") etc.

Reply:
@pawelbaran , I've added an if statement checking validity of attribute